app/helpers/contacts_helper.rb

```diff
@@ -9,4 +9,11 @@ def start_a_conversation_link(aspect, contacts_size)
                   title: t("contacts.index.start_a_conversation")
     end
   end
+
+  def gon_load_contact(contact)
+    Gon.preloads[:contacts] ||= []
+    if Gon.preloads[:contacts].none? {|stored_contact| stored_contact[:person][:id] == contact.person.id }
```

Comment:
`contact.person_id`, unlikely not already loaded here, but just in general, don't load an association just to get at its id.